src/transformers/modeling_utils.py

```diff
@@ -2255,7 +2265,9 @@ def save_pretrained(
 
         hf_quantizer = getattr(self, "hf_quantizer", None)
         quantization_serializable = (
-            hf_quantizer is not None and isinstance(hf_quantizer, HfQuantizer) and hf_quantizer.is_serializable
+            hf_quantizer is not None
+            and isinstance(hf_quantizer, HfQuantizer)
+            and hf_quantizer.is_serializable(safe_serialization=safe_serialization)
```

Comment:
hmmm even though the HfQuantizer classes are not public maybe we should stay on the safe zone and not modify the propery `is_serializable` and keep it as a property. What about creating a new property `is_safe_serializable`, that by default points to `is_serializable` and we overwrite it only for quanto quantizer?

Reply:
Sounds good ! I'll revert the changes I did for the other quantizer and create a new property as you suggested !